src/main/java/com/example/model/User.java

```diff
@@ -0,0 +1,22 @@
+package com.example.model;
+
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Entity
+@Data
+@NoArgsConstructor
+public class User {
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
```

Comment:
to stąd spada, a klasa powinna dziedziczyć po BaseEntity

Reply:
Właściwie to jedna z tych klas powinna być chyba DTO więc bez Entity

Reply:
poprawione, usunięte